webpack/ForemanTasks/Components/TasksDashboard/__tests__/TasksDashboardActions.test.js

```diff
@@ -13,11 +13,17 @@ jest.mock('../TasksDashboardHelper');
 
 const correctTime = 'H24';
 const wrongTime = 'H25';
+const parentTaskID = 7;
 
 timeToHoursNumber.mockImplementation(arg => arg);
 API.get.mockImplementation(async path => {
   if (path === `/foreman_tasks/tasks/summary/${correctTime}`) {
     return { data: 'some-data' };
+  } else if (
+    path ===
+    `/foreman_tasks/tasks/summary/${parentTaskID}/sub_tasks/${correctTime}`
```

Comment:
would it make sense to move all of the consts above into a `fixtures` file which will make it cleaner?
we could also use the same fixtures in the other tests if needed.

Reply:
Added a new file, thanks